key.core/src/main/resources/de/uka/ilkd/key/proof/rules/seqRules.key

```diff
@@ -241,6 +253,17 @@
         \heuristics(simplify_enlarging)
     };
 
+    \lemma
+    getOfSeqUpd {
+        \schemaVar \term Seq seq;
+        \schemaVar \term int idx, jdx;
+        \schemaVar \term any value;
+
+        \find(alpha::seqGet(seqUpd(seq, idx, value), jdx))
+        \replacewith(\if(0<=jdx & jdx < seqLen(seq) & idx=jdx) \then((alpha)value) \else(alpha::seqGet(seq, jdx)))
```

Comment:
```suggestion
        \replacewith(\if(jdx>=0 & jdx <= -1 + seqLen(seq) & jdx=idx) \then((alpha)value) \else(alpha::seqGet(seq, jdx)))
```
I think we could save some rule applications for normalization (~10 per application of the taclet) here if we write down the (in-)equations already in normalized form. On the contrary, the taclet gets less readable. Not sure what we want to have here ...

Reply:
I would favour to keep it in the readable form. KeY is an interactive TP. Normalising as early as possible does not make things more comprehensible at all. We can afford these ten rule apps.